cupy/core/fusion.py

```diff
@@ -228,6 +228,12 @@ def __nonzero__(self):
     def __bool__(self):
         raise Exception("Can't cast to bool")
 
+    def __setitem__(self, slices, value):
+        print(slices)
+        assert slices is Ellipsis or (isinstance(slices, slice) and
```

Comment:
Raise readable error message that represents it only supports Ellipsis or none slice.